switchboard/src/routes/jobs.rs

```diff
@@ -0,0 +1,139 @@
+use crate::auth::db::DbAuth;
+use crate::auth::extract::AuthSource;
+use crate::auth::AuthorizationSource;
+use crate::perms::{ReadJobStatusError, StopJobError, SubmitJobError};
+use crate::routes::proxy::{proxy_err, proxy_val, Proxied};
+use crate::serve::AppState;
+use crate::{impl_from_auth_err, perms, sql};
+use axum::extract::{Path, State};
+use axum::Json;
+use futures_util::stream::FuturesOrdered;
+use futures_util::StreamExt;
+use std::str::FromStr;
+use treadmill_rs::api::switchboard::jobs::{
+    list::Response as LJResponse,
+    status::Response as JSResponse,
+    stop::Response as SJResponse,
+    submit::{Request as EJRequest, Response as EJResponse},
+};
+use uuid::Uuid;
+
+// -- submit
+
+impl_from_auth_err!(EJResponse, Database => Internal, Unauthorized => Unauthorized);
```

Comment:
```suggestion
impl_from_auth_err!(EnqueueJobResponse, Database => Internal, Unauthorized => Unauthorized);
```

We should avoid using abbreviations in types.